openedx/core/djangoapps/credentials/tests/test_signals.py

```diff
@@ -122,6 +123,15 @@ def test_send_grade_records_enabled(self, _mock_is_course_run_in_a_program, mock
         site_config.site_values['ENABLE_LEARNER_RECORDS'] = False
         site_config.save()
         send_grade_if_interesting(self.user, self.key, 'verified', 'downloadable', None, None)
+
+    def test_send_grade_records_disabled_globally(
+        self, _mock_is_course_run_in_a_program, mock_send_grade_to_credentials,
+        _mock_is_learner_issuance_enabled
+    ):
+        self.assertTrue(is_learner_records_enabled())
+        with override_settings(FEATURES={"ENABLE_LEARNER_RECORDS": False}):
+            self.assertFalse(is_learner_records_enabled())
+            send_grade_if_interesting(self.user, self.key, 'verified', 'downloadable', None, None)
         self.assertFalse(mock_send_grade_to_credentials.delay.called)
```

Comment:
Maybe just not seeing well in github, but was this line taken from the test above, rather than copied?

Reply:
Yes indeed! I'll put it back in.